### DISCUSSION TITLE
Production ready Kafka cluster

### DISCUSSION THREAD

**Discussion by dkoci** [strimzi/strimzi-kafka-operator#8456] [Q&A]
Hi.
Is there any instructions how to deploy and configure production ready Kafka cluster with Strimzi operator?

Thank you

**Comment by scholzj** [comment]
A lot depends on your requirements and your environment. We created a video series about things useful for making your cluster production ready which you can check out: https://www.youtube.com/playlist?list=PLpI4X8PMthYeCSpy9a-mGtvDbMgqGNYNy ... it covers some of the important things around storage, rack awareness, etc.
